lib/Command/Turn/Add.php

```diff
@@ -98,6 +98,19 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 			$servers = [];
 		}
 
+		//Checking if the server is already added
+		foreach ($servers as $existingServer) {
+			if (
+				$existingServer['server'] === $server &&
+				$existingServer['protocols'] === $protocols &&
+				$existingServer['secret'] === $secret
```

Comment:
*Question:*
Should also error if the same server+protocol exists with a different secret?

**Must:**
Should also check the schema?